### PR TITLE
Ticketing system new

### PR DESCRIPTION
**Pull request by Bharani0012**
Added new tests case for create custom field api

updated list custom field with pagination functionality

### CODE REVIEW

File: .github/workflows/integrationtests.yml [hunk 1]
```diff
@@ -101,6 +101,7 @@ jobs:
         working-directory: ${{ github.workspace }}
         run: |
           python -m unittest discover -s ./test/IntegrationTests
```
Comment (SaravanakumarR2018) [inline .github/workflows/integrationtests.yml:103]: Only this command does not work, for both the directories?

python -m unittest discover -s ./test/IntegrationTests
 try adding empty __init__.py file within TicketService and check whether the testcases are run without explicitly calling python -m unittest discover -s ./test/IntegrationTests/TicketService
 
 Try to find a way to enable the files under the sub directories to execute without explicitly calling it


Leave this comment open, dont resolve it

Just update the comment with your comments, after you fix it

File: test/IntegrationTests/TicketService/test_create_custom_fields.py [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+    def test_add_custom_field_boolean(self):
+        self._test_add_custom_field("BOOLEAN")
+
+    def test_add_custom_field_date(self):
+        self._test_add_custom_field("DATE")
+
+    def test_add_custom_field_mediumtext(self):
+        self._test_add_custom_field("MEDIUMTEXT")
+
```
Comment (SaravanakumarR2018) [inline test/IntegrationTests/TicketService/test_create_custom_fields.py:203]: Can we add for all these fields?

["VARCHAR(255)", "INT", "BOOLEAN", "DATE", "MEDIUMTEXT", "FLOAT", "TEXT"]

Dont resolve this comment. Just reply to this comment once you fix it

File: test/IntegrationTests/TicketService/test_list_custom_fields.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+            # Log and assert successful creation of each field
+            if response.status_code != HTTPStatus.CREATED:
+                logger.error(f"Failed to add field: {field['field_name']}. Server response: {response.text}")
+            self.assertEqual(response.status_code, HTTPStatus.CREATED,
+                             f"Failed to add field: {field['field_name']}")
+
+        logger.info("Successfully added 50 custom fields.")
+
+    def test_pagination_for_custom_fields(self):
```
Comment (SaravanakumarR2018) [inline test/IntegrationTests/TicketService/test_list_custom_fields.py:130]: Check for getting data for this testcase with page_number = 1 and page_size = 60 and verify only 50 records are replied back

Dont resolve this comment
Just reply to this comment that you have fixed it

File: test/IntegrationTests/TicketService/test_create_custom_fields.py (hunk 2 above)
Comment (Bharani0012) [reply test/IntegrationTests/TicketService/test_create_custom_fields.py:203]: Yes sir we can add all those fields, i included test cases for remaining data types(float, text).

File: test/IntegrationTests/TicketService/test_list_custom_fields.py (hunk 3 above)
Comment (Bharani0012) [reply test/IntegrationTests/TicketService/test_list_custom_fields.py:130]: Resolved sir

File: .github/workflows/integrationtests.yml (hunk 1 above)
Comment (Bharani0012) [reply .github/workflows/integrationtests.yml]: modified sir

**Review by SaravanakumarR2018 — APPROVED** [review]

**Pull request merged**